doc/sheet_functions.md

```diff
@@ -121,6 +121,9 @@ Currently only ETH is supported.</p>
 <dt><a href="#SAN_DAILY_CLOSING_PRICE">SAN_DAILY_CLOSING_PRICE(projectSlug, day)</a> ⇒ <code>number</code></dt>
 <dd><p>Returns the closing price for a given day.</p>
 </dd>
+<dt><a href="#SAN_TOKEN_TOP_TRANSACTIONS">SAN_TOKEN_TOP_TRANSACTIONS(projectSlug, from, to, limit)</a> ⇒ <code>Array</code></dt>
+<dd><p>Top transactions for the token of a given project.</p>
```

Comment:
"Top token transactions for a given slug" sounds better to me

Reply:
You're right, I corrected it now